src/page/product-added-modal.page.ts

```diff
@@ -1,17 +1,18 @@
-import { $, ElementFinder } from 'protractor';
+import { $, ElementFinder, browser } from 'protractor';
+import { protractor } from 'protractor/built/ptor';
```

Comment:
as you are using protractor.ExpectedConditions, will be efficient to import, ExpectedConditions on the first import, and remove this one